docs/download.md

```diff
@@ -30,6 +30,13 @@ bodyclass: download-page
 Note that the main version can contain incompatible changes,
 so please read the changelog carefully when upgrading to it.
 
+### Leaflet Versions Overview
```

Comment:
Please make all headlines like `Using a Hosted Version of Leaflet` a sub-headline of `Leaflet Versions Overview`. So add one `#`

Reply:
I'm generally not convinced this overview is needed. The page is already short and easy to skim headings on, and this only adds unnecessary bloat IMO.